apache-rat-core/src/test/java/org/apache/rat/ReportConfigurationTest.java

```diff
@@ -189,6 +194,19 @@ public void inputFileFilterTest() {
         assertEquals(filter, underTest.getInputFileFilter());
     }
 
+    @Test
+    public void directoryFilterTest() {
+        assertNotNull("Directory filter should not be null by default", underTest.getDirectoryFilter());
+        assertTrue("Directory filter should be a NameBasedHiddenFileFilter one by default", underTest.getDirectoryFilter() instanceof NameBasedHiddenFileFilter);
```

Comment:
When I read this I thought of adding assertj and use its isExactlyInstanceOf(Foo.class) ... not sure if there are reasons to not add assertj-core ;)

Reply:
I didn't want to introduce a new dependency for the test (as assertj is not in our pom). I don't have problem to add assertj and use it if you prefer.

Reply:
I introduced assertj-core and refactore `ReportConfigurationTest` to use it.

If you want, I can refactore all tests to use assertj.